website/docs/sql-reference/sql-distinct.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Is it time to bring back comments + put these together in an other bucket?

Reply:
Im going to avoid hiding these in an "other" folder until we know the sidebar is becoming crowded